mqtt_recorder/recorder.py

```diff
@@ -43,8 +46,20 @@ def __init__(self, host: str, port: int, client_id: str, file_name: str, usernam
         self.__client.connect(host=host, port=port)
         self.__client.loop_start()
 
+    def __csv_writer(self):
+        logger.info('Saving messages to output file')
+        with open(self.__file_name, 'w', newline='') as csvfile:
```

Comment:
Please set this to line buffering

```suggestion
        with open(self.__file_name, 'w', newline='', buffering=1) as csvfile:
```